Rakefile

```diff
@@ -13,8 +13,16 @@ task :check_rng_status do
 
   # RNC must not be newer than RNG
   if rng_commit_time.to_i < rnc_commit_time.to_i
-    raise "Error: control/control.rng is outdated, regenerate it from control/control.rnc file (by running \"make\")"
+    raise "Error: control/control.rng is outdated, regenerate it running \"rake generate_rng\")"
   end
 end
 
+task :generate_rng do
+  unless system("which trang >/dev/null 2>&1")
+    raise "Error: trang package is missing, please install it for proceeding with the conversion."
+  end
+
+  system("trang -I rnc -O rng control/control.rnc control/control.rng")
```

Comment:
for Rake I would prefer to use `sh` instead of system as it provide nicer output. Especially when there is a problem with rnc

Reply:
In case of error it is also reported:

```
rake generate_rng
/home/suse/SUSE/Development/yast-meta/installation-control/control/control.rnc:129:1: error: syntax error
```

VS

```
rake generate_rng
trang -I rnc -O rng control/control.rnc control/control.rng
/home/suse/SUSE/Development/yast-meta/installation-control/control/control.rnc:129:1: error: syntax error
rake aborted!
Command failed with status (1): [trang -I rnc -O rng control/control.rnc co...]
/home/suse/SUSE/Development/yast-meta/installation-control/Rakefile:25:in `block in <top (required)>'
Tasks: TOP => generate_rng
(See full trace by running task with --trace)
```